ALUM/ALUM/Components/FilledInButtonStyle.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  FilledInButtonStyle.swift
+//  ALUM
+//
+//  Created by Yash Ravipati on 1/20/23.
+//
+
+import SwiftUI
+
+struct FilledInButtonStyle : ButtonStyle{
+    @State var disabled: Bool = false
+    
+    func makeBody(configuration: Configuration) -> some View {
+            if(!disabled){
+                configuration.label
+                    .frame(minWidth: 0, maxWidth: .infinity)
+                    .frame(minHeight: 0, maxHeight: .infinity)
```

Comment:
nit I think button height should go till infinity because we want all our buttons to be of consistent size. You can just have this as fixed size if that's easier